From 6cfa52fc6d9f0cdaaad82359655eadcd90d32c93 Mon Sep 17 00:00:00 2001
From: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
Date: Wed, 7 Mar 2018 14:45:08 +0530
Subject: [PATCH 0325/1851] dt: bindings: media: xilinx: Remove unimplemented
 TPG compatible string

commit 3aca4f9620d63efec06f7ebb0a3f05b51d3e721f from
https://github.com/Xilinx/linux-xlnx.git

TPG compatible string "xlnx,v-tpg-6.0" is not supported in the driver.
Hence remove unimplemented compatible string.

Signed-off-by: Radhey Shyam Pandey <radhey.shyam.pandey@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt
index ee65700c9291..f12c244c26a5 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-tpg.txt
@@ -6,7 +6,6 @@ Required properties:
 - compatible: Must contain at least one of
 
     "xlnx,v-tpg-5.0" (TPG version 5.0)
-    "xlnx,v-tpg-6.0" (TPG version 6.0)
     "xlnx,v-tpg-7.0" (TPG version 7.0)
 
   TPG versions backward-compatible with previous versions should list all
@@ -40,7 +39,7 @@ synthesized with two ports and forbidden when synthesized with one port.
 Example:
 
 	tpg_0: tpg@40050000 {
-		compatible = "xlnx,v-tpg-6.0", "xlnx,v-tpg-5.0";
+		compatible = "xlnx,v-tpg-5.0";
 		reg = <0x40050000 0x10000>;
 		clocks = <&clkc 15>;
 
-- 
2.31.1

